pkg/database/mariadb.go

```diff
@@ -59,6 +68,16 @@ func (m *MariaDB) CreateFullBackup(path string) (bp LogPosition, err error) {
 	if path == "" {
 		return bp, fmt.Errorf("no path given")
 	}
+	defer func() {
+		err = m.toggleSlowQueryLog(slowQeryLogON)
```

Comment:
This does not ensure that the `slow_query_log` is enabled again after the backup. Worst case is that the DB is running without it until the next successful backup.

Reply:
yes that is explicitly done. I rather have no slow logs then not doing backups!

Reply:
Sure, just wanted to remark that we still have a slight risk that we disable the slow query log.